docs/pages/example/hillshade.html

```diff
@@ -0,0 +1,32 @@
+<div id='map'></div>
+<script>
+var map = new mapboxgl.Map({
+    container: 'map',
+    style: 'mapbox://styles/mapbox/outdoors-v9',
+    center: [-74.50, 40],
```

Comment:
let's change the map center to somewhere with ⛰ s. Z11, center: -119.5857, 37.7221 is yosemite valley for example

Reply:
Agreed!